src/MAUI/Maui.Samples/Samples/Geometry/SnapGeometryEdits/SnapGeometryEdits.xaml

```diff
@@ -113,7 +113,7 @@
                                                  Light=White}"
                     HorizontalOptions="Center"
                     MaximumHeightRequest="450"
-                    MaximumWidthRequest="250"
+                    MaximumWidthRequest="300"
                     StrokeShape="RoundRectangle 10"
                     VerticalOptions="Center">
                 <ScrollView>
```

Comment:
On iOS, I wasn't aware there was a scroll view in the snap settings, so close button was off screen. In other samples like DisplayClusters or TakeScreenshot, when we have something in focus, tapping the grayed out MapView closes what was in focus and brings back map interactivity. Or making the snap settings slightly taller might be an easier fix, I think it would still be fit on phones. Definitely a UI/UX nitpick so your choice if you want to address this.